mix.exs

```diff
@@ -4,8 +4,8 @@ defmodule Edeliver.Mixfile do
   def project do
     [
       app: :edeliver,
-      version: "1.7.0",
-      elixir: ">= 1.9.0",
+      version: "1.8.0",
+      elixir: ">= 1.10.0",
```

Comment:
Does this mean that we're dropping support for `1.9.0`? If so should this be a major version upgrade?

Reply:
I saw previous elixir 1.9 upgrade affected patch version, so I decided to do the same
Thinking about compartability table, would be useful

Reply:
that is a breaking change indeed and at the same time since API doesn't change I'm leaning towards incrementing minor version, although might be completely wrong on this one

Reply:
I suppose that we may do the same as elixir team: they have dedicated branch for dedicated version. https://github.com/elixir-lang/elixir/branches
Do you like the idea?

Reply:
I guess it addresses a different concern - master branch in Elixir still has all the changes from version branches.